### PR TITLE
add asapdiscovery package

### PR DESCRIPTION
**Pull request by hmacdope**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by hmacdope** [comment]
@jthorton @JenkeScheen @mikemhenry @ijpulidos would you be maintainers on this recipe? Leave a comment saying yes below if so.

**Comment by JenkeScheen** [comment]
yes

**Comment by ijpulidos** [comment]
Yes!

**Comment by mikemhenry** [comment]
Yes

**Comment by hmacdope** [comment]
@condda-forge/help-python could I grab a review here?

**Comment by hmacdope** [comment]
@conda-forge/help-python could I grab a review here?

**Comment by hmacdope** [comment]
@conda-forge/help-python could I grab a review here?

File: recipes/asapdiscovery/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    - pytest-console-scripts
+    - pytest-timeout
+    - requests-mock
+    - mock
+
+    # cli
+    - click
+
+test:
+  imports:
```
[the remark this answers is not in the document]
Comment (hmacdope) [reply recipes/asapdiscovery/meta.yaml:115]: Good idea.

File: recipes/asapdiscovery/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    - codecov
+    - pytest-console-scripts
+    - pytest-timeout
+    - requests-mock
+    - mock
+
+    # cli
+    - click
+
+test:
```
[the remark this answers is not in the document]
Comment (hmacdope) [reply recipes/asapdiscovery/meta.yaml:114]: When i tested locally it was broken for deps  `rdkit` and  `openbabel`, but can try on runner

[the remark this answers is not in the document]
Comment (hmacdope) [reply recipes/asapdiscovery/meta.yaml:113]: image

[the remark this answers is not in the document]
Comment (hmacdope) [reply recipes/asapdiscovery/meta.yaml:113]: These are both installed, so unsure exactly why pip-check fails them, (CC @mikemhenry?). This was why I left the check out, I can just remove again if you are ok with that.

**Pull request merged**